lighthouse-cli/run.js

```diff
@@ -75,7 +75,7 @@ function getDebuggableChrome(flags) {
     port: flags.port,
     ignoreDefaultFlags: flags.chromeIgnoreDefaultFlags,
     chromeFlags: parseChromeFlags(flags.chromeFlags),
-    logLevel: flags.logLevel,
```

Comment:
I don't think this is possible to hit today. Can we add `--debug` flag support to `cli-flags.js` and `bin.js` :D (would love to still be able to get the same `--verbose` output today from the CLI